### ISSUE TITLE
if user mistypes the "existing" channel name then a new channel is created based on the typo

### ISSUE THREAD

**Issue by andrejonathan** [absa-subatomic/quantum-mechanic#679]
LInk-existing-channel: if the user mistypes the "existing" channel name then a new channel is created based on the typo, no check is done if the channel exists.

**Comment by andrejonathan** [comment]
![image](https://user-images.githubusercontent.com/36840575/52398662-84b53180-2ac2-11e9-96e9-b32c7a3fa5f3.png)

 Mistyped jump-jellyfish instead of jumpy-jellyfish, created a new channel

**Comment by kbristow** [comment]
Yeah, I am not sure what the best way to check this is though. Maybe we
there is a gql query we can run to check.
